### PR TITLE
[setup-env] - get demisto-mock from content github if not exist locally

### PR DESCRIPTION
**Pull request by GuyAfik**
## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-9830

## Description
* Fixed an issue where setup-env would fail when trying to copy demistomock.py locally.

### CODE REVIEW

File: demisto_sdk/commands/setup_env/setup_environment.py [hunk 1]
```diff
@@ -85,13 +90,31 @@ def get_docker_python_path(docker_prefix: str) -> List[str]:
             docker_python_path.append(
                 f"{docker_prefix}/{path.relative_to(CONTENT_PATH.absolute())}"
             )
+
+    if not COMMON_SERVER_PYTHON_PATH.exists():
+        try:
+            common_server_python_contnet = TextFile.read_from_github_api(
```
Comment (dorschw) [inline demisto_sdk/commands/setup_env/setup_environment.py:96]: Does this support the offline mode env var?

Comment (GuyAfik) [reply demisto_sdk/commands/setup_env/setup_environment.py:96]: nope, need to add, good catch

Comment (GuyAfik) [reply demisto_sdk/commands/setup_env/setup_environment.py:96]: will be done in https://github.com/demisto/demisto-sdk/pull/4098

Comment (GuyAfik) [reply demisto_sdk/commands/setup_env/setup_environment.py:96]: fixed in https://github.com/demisto/demisto-sdk/pull/4098

File: demisto_sdk/commands/setup_env/setup_environment.py [hunk 2]
```diff
@@ -85,13 +91,35 @@ def get_docker_python_path(docker_prefix: str) -> List[str]:
             docker_python_path.append(
                 f"{docker_prefix}/{path.relative_to(CONTENT_PATH.absolute())}"
             )
+
     if (
-        f"{docker_prefix}/Packs/Base/Scripts/CommonServerPython"
+        f"{docker_prefix}/Packs/Base/Scripts/CommonServerPython"  # CommonServerPython does not exist locally
```
Comment (ilaner) [inline demisto_sdk/commands/setup_env/setup_environment.py:96]: why it means it don't exist locally?

Comment (GuyAfik) [reply demisto_sdk/commands/setup_env/setup_environment.py:96]: if line 92 produces ValueError silently, then the `CommonServerPython` is likely does not exist.

**Review by ilaner — APPROVED** [review]

**Pull request merged**